entry/entrypoint.go

```diff
@@ -68,6 +74,28 @@ func shards(s string) ([]datastor.ShardConfig, error) {
 	return shards, nil
 }
 
+func toTlogConfig(s string) (config.TLog, error) {
```

Comment:
This should instead return a point `*config.TLog` so the check for empty string can return `nil, nil` hence u don't set anything on the config.